README.md

```diff
@@ -181,6 +183,7 @@ $ jest src/test/{test-name}.js
   * **`router.js`** - Routes app URLs to controllers
   * **`ssltunnel.js`** - Utilities to determine state of tunnel and manage the PageKite process
   * **`tunnel_setup.js`** - Express middleware to determine if the tunnel is set up
+  * **`user-profile.js`** - Manages persistent user data
```

Comment:
I'm not sure it's really a user profile if it's shared by all users, but to be honest I can't think of a better name.